Import selected AI deployment sections (#97)
SUSE/doc-suse-ai: tasks/AI-gpu-nodes-assigning.adoc

include::../common/generic-attributes.adoc[]
// overridden title?
ifdef::override-title[]
= {override-title}
endif::[]
ifndef::override-title[]
= Assigning GPU resources to applications
endif::[]
// overridden abstract?
ifdef::override-abstract[]
{override-abstract}
endif::[]
ifndef::override-abstract[]
Applications running on {kube} can request GPU resources when GPU acceleration is required.

After the GPU Operator is installed and configured, GPU resources become available to {kube}.
Applications request these resources by defining resource requests and limits for their workloads.

{kube} schedules workloads that request GPU resources on nodes where the requested resources are available.
Optionally, you can use node selection rules to restrict an application to specific nodes.
endif::[]
// erase the flag for future overrides

:override-abstract!:
:override-title!:

:revdate: 2026-08-25
:page-revdate: {revdate}

.**Requirements**

* {rke2a} {kube} cluster with one or more nodes that provide supported GPU resources.

> * A GPU Operator must be installed and configured so that the GPU resources are available to {kube}.

Reviewer: ```suggestion
* A GPU Operator must be installed and configured to make GPU resources available to {kube}.
```
